src/devices/bus/coco/coco_multi.cpp

```diff
@@ -178,6 +180,7 @@ static void coco_cart_slot4(device_slot_interface &device)
 	device.option_add("rs232", COCO_RS232);
 	device.option_add("dcmodem", COCO_DCMODEM);
 	device.option_add("orch90", COCO_ORCH90);
+	device.option_add("stereo_composer", COCO_STEREO_COMPOSER);
```

Comment:
Does the name here need to be this long?  Most of the options here use terse, abbreviated names for the command line.